src/router.ts

```diff
@@ -17,30 +20,30 @@ const routes:RouteRecordRaw[] = [
               : next({path: '/welcome'}))
           : next()
     },
-    component: () => load('layouts/default/Default.vue'),
```

Comment:
This undoes the lazy-loading you implemented previously?

Reply:
@mlincoln - It does. Performance-wise, it's a one time cost when the site first loads in the browser.  All components are loaded immediately.  If this works then at least I've identified the problem and can decide how much we want lazy-loading